src/views/GameDetails/GameServer/GameServer.scss

```diff
@@ -0,0 +1,101 @@
+@import "../../../index";
+
+.template-title {
+    color: $color-white;
+    align-self: center;
+    font-size: 2rem
+}
+
+.server-container {
+    display: flex;
+    justify-content: space-evenly;
+    
+    div {
+        min-width: 400px;
+        display: flex;
+        flex-direction: column;
+    }
+
+    .input-field {
+        display: flex;
+        flex-direction: row;
+
+        input {
+            padding: 20px;
+            width: 70%;
+            font-size: 1rem;
+            border: none;
+            border-radius: 4px 0 0 4px;
+            box-shadow: 6px 6px 14px -5px rgba(0, 0, 0, 0.5);
+            background-color: transparent;
+            color: white;
+        }
+    }
+
+    .game-title {
+        color: $color-white;
+        h1 { font-size: 100%; }
+        h2 { font-size: 120%; }
+    }
+}
+
+.game-name {
+    line-height: 20rem;
+}
+
+.game-select {
+    color: $color-white;
+
+    #slider {
+        -webkit-appearance: none;
+        width: 20%;
+        height: 5px;
+        outline: none;
+        border-radius: 3px;
+    }
+
+    .range {
+        height: 40px;
+        width: 100%;
+        border-radius: 10px; 
+    }
+
+    .range .slider-value {
+        position: relative;
+        width: 100%;
+    }
+
+    .range .field {
+        position: relative;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        height: 100%;
+    }
+
+    .range .field .value {
+        position: absolute;
+        font-size: 18px;
+        font-weight: 600;
+    }
+
+    .range .field .value.left {
+        left: -22px;
+    }
+
+    .range .field .value.right {
+        right: -43px;
+    }
```

Comment:
These styles don't seem to change anything. I noticed that your class names in `GameServer.js` were `value-left` and `value-right`, so `value.left` and `value.right` won't work here. Also, `field` has no descendant called `value` so those styles are discarded as well. I viewed this on both Firefox and Chrome, so let me know if you saw different results.

Reply:
Removed all of it. Nothing changed. Initially had these values set for something else that I ended up deleting later